### DISCUSSION TITLE
peak2gene correlation

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#1210] [Questions / Documentation]
Hi @rcorces  and @jgranja24 ,

The peak2gene correlation matrix gives me some unexpected results, and I was wondering if you have any suggestions on how to fix it?

FOXC1 is a known TF regulator of NRG1 gene. You can see that NRG1 and FOXC1 inferred expressions are also co-locally expressed(where I expected them to be expressed). So technically I expected to find a peak with FOXC1 binding site that would highly be correlated with NRG1 expression

However, when I do the peak2gene correlation analysis I don't even see any peak with a high correlation with NRG1 inferred expression, which is a bit odd.

<img width="269" alt="Screen Shot 2021-12-02 at 7 32 33 PM" src="https://user-images.githubusercontent.com/19545198/144540635-bd42326a-2f96-4437-a526-975ca452082c.png">
<img width="645" alt="Screen Shot 2021-12-02 at 7 36 43 PM" src="https://user-images.githubusercontent.com/19545198/144541023-1613a1e0-5448-4605-9537-61ab7e4360ce.png">

**Comment by rcorces** [comment]
> FOXC1 is a known TF regulator of NRG1 gene. You can see that NRG1 and FOXC1 inferred expressions are also co-locally expressed(where I expected them to be expressed). So technically I expected to find a peak with FOXC1 binding site that would highly be correlated with NRG1 expression. However, when I do the peak2gene correlation analysis I don't even see any peak with a high correlation with NRG1 inferred expression, which is a bit odd.

I'm not sure what to say other than the absence of evidence isnt the evidence of absence. If you look by eye at tracks, do you see clear peaks that you think should have high correlation?
